### PR TITLE
README: add Forum

### PR DESCRIPTION
**Pull request by Andersson007**
Dear maintainers,
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README and the docsite. Similar PRs are now being raised across all the Ansible ecosystem projects.

Thank you

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @oraNod 

ready for review

File: docs/README.md [hunk 1]
```diff
@@ -261,7 +261,41 @@ to get started with developing the extension.
...
+  fellow enthusiasts.
+- [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide
+  announcements including social events. The
+  [Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn),
+  which is used to announce releases and important changes, can also be found
+  here.
+
+For more information on the forum navigation, see
+[Navigating the Ansible forum](https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39)
+post.
```
Comment (oraNod) [inline docs/README.md:291]:
```suggestion
See `Navigating the Ansible forum <https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39>`_ for some practical advice on finding your way around.
```

**Comment by Andersson007** [comment]
@oraNod the suggestion has been committed, PTAL

**Review by oraNod — APPROVED** [review]
LGTM. Thanks @Andersson007

**Review by gundalow — APPROVED** [review]

**Review by ssbarnea — APPROVED** [review]

**Pull request merged**